manifests/puppet.pp

```diff
@@ -16,8 +16,8 @@
 class simp_options::puppet (
   Simplib::Host $server,
   Simplib::Host $ca,
-  Simplib::Port $ca_port = 8141,
-  Simplib::Serverdistribution $server_distribution = 'PC1'
+  Simplib::Serverdistribution $server_distribution = fact('pe_build') ? { Undef => 'PC1', default => 'PE' },
```

Comment:
Now we're changing defaults. It shouldn't be an issue either, but it adds complexity to this PR.

Reply:
@jeefberkey The default for PE was set in error. This is appropriate.

@trevor-vaughan `$facts['is_pe']`, which is part of stdlib since 2.5.0, would be better here.

Reply:
@lamawithonel Hmm...Ok, I dug through and didn't see that so I followed what was in `pupmod`. I'll get it changed.